README.md

```diff
@@ -43,6 +43,9 @@ Contributers can build El-MAVEN on Windows, Ubuntu or Mac systems by following t
 - `pacman --force -Syu`
 - `pacman --force -Su`
 - `pacman --force -Sy base-devel msys2-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-qt5 mingw64/mingw-w64-x86_64-hdf5 mingw64/mingw-w64-x86_64-netcdf mingw64/mingw-w64-x86_64-boost msys/git mingw-w64-x86_64-curl`
+-  `Download OpenSSL package using https://indy.fulgan.com/SSL/openssl-1.0.2r-x64_86-win64.zip`
```

Comment:
Since this is not a command, can we write this as a regular instruction instead. Like:
> Download OpenSSL package using `https://indy.fulgan.com/SSL/openssl-1.0.2r-x64_86-win64.zip`

Alternatively, we could have a `curl` command in place for the equivalent effect instead of instructing them to download.